package.json

```diff
@@ -232,7 +229,6 @@
     "react-addons-perf": "^15.3.0",
     "react-autosuggest-ie11-compatible": "^8.0.6",
     "react-cropper": "^1.0.1",
-    "react-datepicker": "^0.29.0",
```

Comment:
Was this the only module unique to these apps? Just wondering if there could be any more more dependencies that we could remove.

Reply:
I tried a quick run of `depcheck` but it gave me a bunch of false positives. I think there are a couple of other utilities we could try.

Reply:
This was the only one I found when updating React deps for another PR.